doc/user_manual/chSpecialElements.tex

```diff
@@ -765,8 +763,8 @@ \subsection{Additional Flags}
 
 \subsection{Output File \texttt{dynksets.dat}}
 
-When a \texttt{DYNK} block is present in the input file, a file \texttt{dynksets.dat} is created and in the current working directory.
-Unless a \texttt{NOFILE} statement is present, this file contains first a header
+When the \texttt{DYNKSETS} flag is present in the \texttt{DYNK} block, a file \texttt{dynksets.dat} is created and in the current working directory.
```

Comment:
Keep a mention of what the meaning of "NOFILE" was, e.g. a sentence "The DYNKSETS flag was introduced in version xx.xx.xx, replacing the NOFILE flag which had the opposite effect."

Reply:
Thanks!